src/components/blockquote-block/blockquote-block.vue

```diff
@@ -49,6 +50,10 @@
       linkIsExternal: {
         type: Boolean,
         default: false
+      },
+      slug: {
```

Comment:
Can we call the `slug` prop `titleId`? Then the prop name is more explicitly describing what it's used for. I think slug is a bit confusing since we use it a lot for url params.